src/util/debug.h

```diff
@@ -195,6 +195,8 @@ void sss_debug_fn(const char *file,
                                                       SSSDBG_CRIT_FAILURE))))
 
 
+#define DEBUG_CHAIN_ID_FMT "[RID#%lu] "
```

Comment:
Is this expected to be used outside of `debug.c`?

Reply:
No, do you propose to change it to just `CHAIN_ID_FMT` ?

Reply:
I propose to keep it in `debug.c` then...

Reply:
Fixed